.github/workflows/nightly_Linux_distributions.yml

```diff
@@ -37,6 +37,6 @@ jobs:
     - name: build and compile
       run: |
         mkdir build && cd build
-        cmake $CMAKE_FLAGS -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DBUILD_SHARED_LIBS=${{ matrix.shared_libraries }} -DCMAKE_CXX_COMPILER=${{ matrix.compiler }} ..
+        cmake $CMAKE_FLAGS -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DBUILD_SHARED_LIBS=${{ matrix.shared_libraries }} -DCMAKE_CXX_COMPILER=${{ matrix.compiler }} -DEXIV2_ENABLE_PNG=ON -DEXIV2_ENABLE_CURL=ON -DEXIV2_BUILD_UNIT_TESTS=OFF -DEXIV2_ENABLE_BMFF=ON -DCMAKE_INSTALL_PREFIX=install ..
```

Comment:
AFAIK, -DEXIV2_ENABLE_CURL=ON has no effect unless -DEXIV2_ENABLE_WEBREADY=ON is also specified (OFF by default):

https://github.com/Exiv2/exiv2/blob/33d38dc4724b5403dc3965631755d794ad1119ca/cmake/generateConfigFile.cmake#L7

Reply:
Good catch. I'll enable then WEBREADY too (and check that the distributions are installing the required dependencies)

Reply:
It turns out we were using already the right options in this file with the variable `CMAKE_FLAGS`.